### DISCUSSION TITLE
Windows fastboot not compatible with Grub

### DISCUSSION THREAD

**Discussion by stedio** [badstorm/surface-pro-7-opencore#45] [Q&A]
The annoying red warning bar is removed with Grub, however it also makes windows fastboot not working, directly getting back to login window when click shutdown from start menu. Disable windows fastboot solves the problem. 

Any solution to enable fastboot?

**Comment by badstorm** [comment]
I think this is not a issue of the project, but it  related to Grub. I move this question to Discussions
